dot/rpc/modules/state.go

```diff
@@ -489,6 +495,41 @@ func (sm *StateModule) QueryStorage(
 	return nil
 }
 
+// QueryStorageAt queries historical storage entries (by key) at the best block if the given block hash is nil
+func (sm *StateModule) QueryStorageAt(
+	_ *http.Request, req *StateStorageQueryAtRequest, res *[]StorageChangeSetResponse) error {
+	var atBlockHash common.Hash
+	if req.At.IsEmpty() {
+		atBlockHash = sm.blockAPI.BestBlockHash()
+	} else {
+		atBlockHash = req.At
+	}
```

Comment:
nit a tiny bit shorter

```suggestion
	atBlockHash := req.At
	if atBlockHash.IsEmpty() {
		atBlockHash = sm.blockAPI.BestBlockHash()
	}
```